README.md

```diff
@@ -171,7 +174,29 @@ tox -e mlflow-remote
 tox -e mlflow-local
 ```
 
-#### Include NVIDIA GPU UATs
+### NVIDIA GPU UAT
+
+#### Run NVIDIA GPU UAT from inside a notebook
+In order to run the NVIDIA GPU UAT from inside a notebook, follow the same steps described in the [From inside a notebook](#running-inside-a-notebook) section above. However, make sure to follow the prerequisites if needed.
```

Comment:
We should move the however sentence before the link otherwise I have hard time to follow correct instructions.